### ISSUE TITLE
version 0.16.0 UtilityKeywordExtensions.ToSql NullReferenceException

### ISSUE THREAD

**Issue by OdaShinsuke** [Codeer-Software/LambdicSql#1]
Reproduction code

```CS
using System;
using LambdicSql;
public class DB { }
public class Hoge
{
    public int Value1 { get; set; }
    public DateTime Value2 { get; set; }
}
class Program
{
    static void Main(string[] args)
    {
        Console.WriteLine(Db<DB>.Sql(db => "{0}".ToSql(new Hoge { Value1 = Int32.MaxValue } )));
        Console.WriteLine(Db<DB>.Sql(db => "{0}".ToSql(new Hoge { Value2 = new DateTime(2017, 1, 12) })));
        var nullexception = Db<DB>.Sql(db => "{0}".ToSql(new Hoge { Value2 = DateTime.MaxValue })); // NullReferenceException!
    }
}
```

**Comment by Ishikawa-Tatsuya** [comment]
Thank you.
A null exception occurs when a static property is used for a parameter.
I fixed a bug and released 0.18.0.

**Comment by Ishikawa-Tatsuya** [comment]
bug fixed.
